app/models/search.rb

```diff
@@ -0,0 +1,26 @@
+class Search
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+  include ActiveModel::Validations
+
+  attr_accessor :query
+
+  validates :query, presence: true
+
+  def persisted?
+    false
+  end
+
+  def initialize(params = {})
+    @query = params[:query]
+  end
+
+  def results
+    # RAILS4: Return PgSearch::Document.none when no query
+    PgSearch.multisearch(query).where("? >= publish_at", Time.current) if query
```

Comment:
Would have to expand Pg search to get a scope, I think that would be to much work for a single where condition